models.py

```diff
@@ -672,6 +682,8 @@ class MerchantTxModelView(BaseOnlyUserOwnedModelView):
     column_default_sort = ('date', True)
     column_exclude_list = ['user', 'wallet_address']
     column_formatters = {'amount':_format_amount, 'direction':_format_direction}
```

Comment:
with the above changes you will need a column formatter for 'amount_nzd', use the same '_format_amount' function and update it to handle name = 'amount_nzd'